spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/repository/support/SpannerRepositoryImplTests.java

```diff
@@ -66,8 +82,20 @@
 	public void setup() {
 		this.template = mock(SpannerTemplate.class);
 		this.entityProcessor = mock(SpannerEntityProcessor.class);
+		this.mappingContext = mock(SpannerMappingContext.class);
```

Comment:
Correspondingly, after SpannerTemplate.existsById method is implemented, these tests should be moved to `SpannerTemplateTests` class.